src/content/advanced/cluster-management/node-pools-capi/index.md

```diff
@@ -0,0 +1,322 @@
+
+On AWS EC2 (CAPA) you can override the instance type of the node pool to enable [mixed instances policy on AWS](https://docs.aws.amazon.com/autoscaling/ec2/APIReference/API_LaunchTemplateOverrides.html).
+This can provide Amazon EC2 Auto Scaling with a larger selection of instance types to choose from when fulfilling node capacities.
+
+You can set the `instanceTypeOverrides` value when defining your node pool in the cluster values. For example:
+
+```yaml
+metadata:
+  description: "my cluster"
+  name: test-cluster
+  organization: giantswarm
+nodePools:
+  nodepool0:
+    maxSize: 4
+    minSize: 3
+    instanceTypeOverrides:
+    - r6i.xlarge
+    - r5.xlarge
+    - m5.xlarge
+```
+
+Please notice that when setting `instanceTypeOverrides`, the `instanceType` value will be ignored, and the instance types defined in `instanceTypeOverrides` will be used instead.
+Also, the order in which we define the instance types in `instanceTypeOverrides` is important.
+The first instance type in the list that is available in the selected availability zone will be used.
+
+**Note:** Changing the `instanceTypeOverrides` value won't trigger a rolling update of the node pool.
+
+Using multiple instance types in a node pool has some benefits:
+
+- Together with spot instances, using multiple instance types allows better price optimization. Popular instance types tend to have more price adjustments. Allowing older-generation instance types that are less popular tends to result in lower costs and fewer interruptions.
+
+- Even without spot instances, AWS has a limited number of instances per type in each Availability Zone. It can happen that your selected instance type is temporarily out of stock just in the moment you are in demand of more worker nodes. Allowing the node pool to use multiple instance types reduces this risk and increases the likelihood that your node pool can grow when in need.
+
+## Node pools and autoscaling {#autoscaling}
+
+With node pools, you set the autoscaling range per node pool (suppported on AWS EC2 (CAPA) clusters only). The Kubernetes cluster autoscaler has to decide which node pool to scale under which circumstances.
+
+If you assign workloads to node pools as described [above](#assigning-workloads) and the autoscaler finds pods in `Pending` state, it will decide based on the node selectors which node pools to scale up.
+
+In case there are workloads not assigned to any node pools, the autoscaler may pick any node pool for scaling. For details on the decision logic, please check the upstream FAQ for [AWS](https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/cloudprovider/aws/README.md).
+
+## On-demand and spot instances {#on-demand-spot}
+
+{{< tabs >}}
+{{< tab id="cluster-capa-ec2" for-impl="capa_ec2" >}}
+
+Node pools can make use of [Amazon EC2 Spot Instances](https://aws.amazon.com/ec2/spot/). You just need to enable it and select the max price to pay on the node pool definition.
```

Comment:
```suggestion
Node pools can make use of [Amazon EC2 Spot Instances](https://aws.amazon.com/ec2/spot/). On the node pool definition, you can enable it and select the maximum price to pay.
```

Avoiding the word "just" because it can also relay a passive-aggressive "Hey it's so easy, a baby could do it" meaning.